src/main/java/com/thomashan/coup/player/Players.java

```diff
@@ -1,27 +1,37 @@
-package com.thomashan.coup;
+package com.thomashan.coup.player;
 
 import java.util.List;
 
 public interface Players {
+    static Players create(Player... players) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'UR'-anomaly for variable 'player' (lines '6'-'10').](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)